packages/web-app-files/src/views/trash/Overview.vue

```diff
@@ -175,7 +174,27 @@ export default defineComponent({
     ])
 
     const getSpaceName = (space: SpaceResource) => {
-      return isPersonalSpaceResource(space) ? $gettext('Personal') : space.name
+      if (isPersonalSpaceResource(space)) {
+        return $gettext('Personal')
+      }
+      if (space.disabled !== false) {
```

Comment:
```suggestion
      if (space.disabled === true) {
```

easier to read :-) and this way the absence of the `disabled` prop in the space object would result in the space being shown as enabled. Which sounds correct to me.